src/whiteboard/manager.cpp

```diff
@@ -869,6 +873,9 @@ bool manager::save_recruit(const std::string& name, int side_num, const map_loca
 			created_planned_recruit = true;
 
 			print_help_once();
+#else
+			created_planned_recruit = false;
```

Comment:
It wasn't a macro I knew about, but it's nice that it's included in global.hpp. Have added this now, hope it will resolve the blockage.

As for the work-around itself, still don't know if it's a good idea as bug #24454 suggests at least some people are still using recruit planning.